### ISSUE TITLE
Exception in native call from js

### ISSUE THREAD

**Issue by hitesh905** [pusher/pusher-websocket-react-native#152]
i'm getting this error when pusher?.unsubscribe methods invokes. any idea how to resolve this issue ?

<img width="400" alt="Screenshot 2024-03-22 at 10 31 43 AM" src="https://github.com/pusher/pusher-websocket-react-native/assets/140825658/74e5940e-df21-415f-9bcb-56d3e3506f88">
<img width="515" alt="Screenshot 2024-03-22 at 10 31 57 AM" src="https://github.com/pusher/pusher-websocket-react-native/assets/140825658/c7dce8f1-ceee-485a-a7d5-6e5cea7bb76b">
<img width="1512" alt="Screenshot 2024-03-22 at 10 32 02 AM" src="https://github.com/pusher/pusher-websocket-react-native/assets/140825658/2c5e552c-583f-44ad-aec1-3dd5c58cd7be">

**Comment by benw-pusher** [comment]
Are you able to retrieve and share the full error trace from the device? this should contain more info about the exact error encountered

**Comment by hitesh905** [comment]
Yes while unsubscribing to channel i'm getting this. Please check attached screenshot for stack trace.
<img width="1413" alt="Screenshot 2024-03-26 at 10 55 08 AM" src="https://github.com/pusher/pusher-websocket-react-native/assets/140825658/b962a479-3196-4254-9c73-844edf22d996">
